src/Elasticsearch.Net/Api/RequestParameters/RequestParameters.NoNamespace.cs

```diff
@@ -361,13 +361,6 @@ public class DeleteByQueryRequestParameters : RequestParameters<DeleteByQueryReq
 			set => Q("analyze_wildcard", value);
 		}
 
-		///<summary>The analyzer to use for the query string</summary>

```

Comment:
beaking change?

Reply:
yes, looks like it was removed from the spec: 

- 7.3: https://github.com/elastic/elasticsearch/blob/7.3/rest-api-spec/src/main/resources/rest-api-spec/api/delete_by_query.json#L28-L31
- 7.4: https://github.com/elastic/elasticsearch/blob/7.4/rest-api-spec/src/main/resources/rest-api-spec/api/delete_by_query.json